### DISCUSSION TITLE
feat: zstack function

### DISCUSSION THREAD

**Discussion by Onekki** [chakra-ui/panda#1511] [Ideas]
### Description

A extension to the existing stack function. Instead of stacking the items vertically or horizontally it stacks the children on top of each other.

### Problem Statement/Justification

Currently there exists no function that lets its children stack on top of each other

### Proposed Solution or API

Add absolute position for children and relative position for parent.

**Comment by astahmer** [comment]
not sure if many people would use this, I feel like this should remain in user-land, as a custom preset or utility

**Comment by omarkhatibgg** [comment]
@Onekki I'm curious to see what do you means?
or what is the difference between ZStack and VStack?

**Comment by Onekki** [comment]
> @Onekki I'm curious to see what do you means? or what is the difference between ZStack and VStack?

You can refer to this PR:
https://github.com/chakra-ui/chakra-ui/pull/8039

**Comment by Onekki** [comment]
> not sure if many people would use this, I feel like this should remain in user-land, as a custom preset or utility

I'm from Flutter & SwiftUI, they all have this feature. If this function were available, people would naturally use it.
